### ISSUE TITLE
Mozilla VPN popup = poor OOBE

### ISSUE THREAD

**Issue by szmarczak** [mozilla/multi-account-containers#2589]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

N/A

### Actual behavior

![image](https://github.com/mozilla/multi-account-containers/assets/36894700/c731cd02-6e96-421a-98a0-8559e4c706ff)

### Expected behavior

![image](https://github.com/mozilla/multi-account-containers/assets/36894700/6fd5111c-1e54-4937-874d-ad5216250a51)

### Additional informations

The web is flooded with cookie consent popups. There has been backlash. Mozilla seems to follow a similar route by throwing Mozilla VPN everywhere possible. Having a slide in the onboarding slides is enough, don't force people people to say "no". Make it opt-in, not opt-out (don't force people to interact with popups). If you really insist on having a popup about Mozilla VPN make it so that it doesn't obstruct the actual view underneath.

**Comment by ieure** [comment]
Agreed.  I instantly uninstalled this extension because of it bombarding me with ads.

The model here seems to be to nerf the built-in container support, then lock full functionality away in a confusingly-similarly named extension that tries to upsell you Mozilla's VPN.

I don't think much of that.

**Comment by jimrs** [comment]
agreed, though noone at the corporation cares. the people at the mozilla corp know very well what they are doing. the managers need their powerpoint presentations where a green number will say "mozilla vpn users increased". removing the annoying services suggestions (vpn, relay, pocket, the homepage suggestions, the "features" recommendations that you have to opt out of) would mean red numbers. simple as.
as always, it is up to the users to fix the problems.

**Comment by klint** [comment]
Just click on the check box to remove that part and go back to your life... Or give fund Mozilla with some millions to avoid the need of a VPN and all those features that you find annoying...

**Comment by draeath** [comment]
This is absurd. I'm not going to be deploying this into profiles as-is.

I would love to see an about:config entry or similar to get rid of it.

**Comment by CleyFaye** [comment]
> Just click on the check box to remove that part and go back to your life... Or fund Mozilla with some millions to avoid the need of a VPN and all those features that you find annoying...

This message, and the quite large section, about using Mozilla VPN is present here, and in all container config pages. This happens even when the extension does not have the permission to actually communicate with Mozilla VPN, or when no account is setup.

This is simply a bad user experience. Accepting that "it's what it is" without questioning it just because Mozilla is actually *spending* their money on that feature instead of the gazillion of actual issues is not good.

There is no problem in integrating features when it makes sense. Forcing it onto people is the exact bad practice that pushes people away from other solutions toward Firefox. If the same strategies applies here, people will just move out again.
